CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: Issue with geo_point in polygon query which is bug in Lucene
+   and relates to https://github.com/elastic/elasticsearch/issues/20333
```

Comment:
pls describe the problem that has been fixed from a user perspective, no need to point to the ES issue